File: Back/src/main/java/model/auxiliares/Endereco.java
package model.auxiliares;

import enumerations.UF;
import exports.BaseEntity;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Endereco  extends BaseEntity{

	private String cep;
	private String logradouro;
	private String numero;
	private String cidade;
	private UF uf;
	private String bairro;
	private String complemento;
	
}

File: Front/src/app/exports/interface/endereco.ts
import { Uf } from "../enum/uf";

export interface Endereco {

    cep: string;
    logradouro: string;
    numero: string;
    cidade: string;
    quadra: string;
    lote: string;
    uf: Uf;
    bairro: string;
    complemento: string;

}
